app/ios/Plugin/AppPlugin.swift

```diff
@@ -92,4 +105,8 @@ public class AppPlugin: CAPPlugin {
             ])
         }
     }
+
+    @objc func minimizeApp(_ call: CAPPluginCall) {
+        throw AppPluginError.notImplemented
```

Comment:
iOS code is not compiling because it should be `case .notImplemented:` instead of `switch .notImplemented:` and also complains about the throw.
You should just use the unimplemented call method like this and you can remove the whole `AppPluginError` enum.
```suggestion
        call.unimplemented()
```